app/models/aggregation_filter.rb

```diff
@@ -0,0 +1,7 @@
+class AggregationFilter < ActiveRecord::Base
+  belongs_to :user
+
+  PULL_REQUEST_FILTER = AggregationFilter.where("pull_requests.user_id = aggregation_filters.user_id")
```

Comment:
This would probably be more obvious as a class method on `AggregationFilter`, something like:

```ruby
def self.pull_request_filter
  where("pull_requests.user_id = aggregation_filters.user_id")
  .where("pull_requests.repo_name LIKE aggregation_filters.repo_pattern")
  .exists.not
end
```